roles/cobbler_profile/defaults/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+# Where to download ISOs
+iso_dir: /var/lib/cobbler/isos
+# Mount point to use for ISOs during import
+iso_mount: /mnt/iso
+
+distros:
```

Comment:
seems kinda weird to call this 'distros' when the final result is a profile and the role is called cobbler_profile.